theme/lumo/vaadin-grid-styles.html

```diff
@@ -157,7 +160,7 @@
         background: var(--lumo-primary-color-50pct);
       }
 
-      #scroller:not([ios]) [part~="row"][dragstart]:not([dragstart=""])::after {
+      #scroller [part~="row"][dragstart]:not([dragstart=""])::after {
```

Comment:
The row count number works on iOS 13 iPad pro Safari so we can just re-enable it for all the browsers. No side-effects on iPhone either.